### PR TITLE
Improve Typescript types

### PR DESCRIPTION
**Pull request by DavidRouyer**
- Use the true es6 import syntax: `import bodybuilder from 'bodybuilder'`
- Change the types file structure

### CODE REVIEW

**Comment by ferronrsmith** [comment]
Thanks !

**Comment by DavidRouyer** [comment]
Can you publish a v2.2.12 with thoses changes please?

**Comment by danpaz** [comment]
Done

**Comment by DavidRouyer** [comment]
Thanks!

File: types/test/bodybuilder-tests.ts [hunk 1]
```diff
@@ -1,6 +1,4 @@
-/// <reference path="../bodybuilder.d.ts" />
-
-import bodybuilder = require('bodybuilder');
+import bodybuilder from 'bodybuilder';
```
Comment (johannes-scharlach) [inline types/test/bodybuilder-tests.ts]: If we have to change this, it's a breaking change to the types (see #197). Technically currently both ways of importing work, I think.

I cannot think of a way how both ways of importing will keep on working.

**Comment by ferronrsmith** [comment]
Think we should revert these changes.

**Comment by johannes-scharlach** [comment]
Agree. We can keep the files moved, but should stick with the previous version of the definition file.

**Comment by DavidRouyer** [comment]
It looks like the CommonJS module syntax and the ES6 module syntax are incompatible as per https://www.typescriptlang.org/docs/handbook/modules.html.
Better rolling back to 
```javascript
declare module 'bodybuilder' {
  export = bodybuilder;
}
```
and release a major version with
```javascript
export as namespace bodybuilder;
export default bodybuilder;
```

**Pull request merged**